control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources_test.go

```diff
@@ -2036,6 +2143,19 @@ func TestReconcileAuthOIDC(t *testing.T) {
 					g.Expect(errors.IsNotFound(err)).To(BeTrue())
 				}
 			}
+			// Verify that no unexpected client secrets were copied
+			secretList := &corev1.SecretList{}
+			err = hcClient.List(ctx, secretList, client.InNamespace(ConfigNamespace))
+			g.Expect(err).ToNot(HaveOccurred())
+			expectedSecrets := map[string]struct{}{}
+			for _, name := range test.expectOIDCClientSecrets {
+				expectedSecrets[name] = struct{}{}
+			}
```

Comment:
```suggestion
			expectedSecrets := sets.New(test.expectOIDCClientSecrets...)
```
could use a set instead of map[]struct.